### PR TITLE
Details component

### PR DESCRIPTION
**Pull request by golosgit**
2 components in 1 pull - feature/MovieDetail and feature/PersonDetail.

I added PeopleList to MovieDetail and MoviesList to PersonDetail in its original form. When we will implement logic a lot will change (some info will be added as props or from external API).

In App.js MoviesList has been changes with MovieDetails and PeopleList with PersonDetails so navigation will redirect to movie/person details page so it's easier to test (before merge I will correct it to proper state).

### CODE REVIEW

File: src/features/movies/MovieDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        <PosterWrapper>
+          <MainInfoWrapper>
+            <MainInfoTitle>Mulan long title</MainInfoTitle>
+            <RatingWrapper>
+              <StarIcon infoMain />
+              <Rate infoMain >7,8</Rate>
+              <Votes infoMain >/ 10</Votes>
+              <Votes infoMain normalHidden>335 votes</Votes>
+            </RatingWrapper>            
+            <RatingWrapper mobileHidden>
```
Comment (maxnatalia) [inline src/features/movies/MovieDetails/index.js]: I have error in the consol with `infomain`.

File: src/features/movies/MovieDetails/styled.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+export const PosterWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  margin: 0 auto;
+  padding: 0 16px;
+  max-width: 1368px;
+  height: 769px;
+  background-image: url(${bigPoster});
```
Comment (maxnatalia) [inline src/features/movies/MovieDetails/styled.js:16]: When we don't have a poster the background is entirely black, maybe we should added some effects with shadows?

File: src/features/movies/MovieDetails/styled.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  box-shadow: 
+    inset 0 0 40px 40px rgb(0 0 0),
```
Comment (maxnatalia) [inline src/features/movies/MovieDetails/styled.js:21]: I don't see this shadow...when we don't have the poster

File: src/features/movies/MovieDetails/index.js (hunk 1 above)
Comment (maxnatalia) [reply src/features/movies/MovieDetails/index.js]: For mobile 320px is difference versus Figma (In bigposter, In cast and crew - number of tiles we need 2 columns but now we have one)

Comment (maxnatalia) [reply src/features/movies/MovieDetails/index.js]: For desktop I can see 5 columns with tiles but in Figma we have 6 columns (cast and crew).

File: src/features/people/PersonDetails/index.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+              Liu lived there for four years.
+            </DetailsText>
+          </Info>
+      </DetailsWrapper>
+      <MoviesList />
+      <MoviesList />
+    </MainWrapper>
+  );
+};
+
```
Comment (maxnatalia) [inline src/features/people/PersonDetails/index.js:43]: Here, the stars and rating in mobile 320px is situated in the end of the container but in Figma is different.

**Review by maxnatalia — COMMENTED** [review]
Need to improved some things with display in mobile versions and desktop version in accordance with the comments.

File: src/features/movies/MovieDetails/styled.js (hunk 2 above)
Comment (golosgit) [reply src/features/movies/MovieDetails/styled.js:16]: When there won't be poster we won't display PosterWrapper at all.

File: src/features/movies/MovieDetails/styled.js (hunk 3 above)
Comment (golosgit) [reply src/features/movies/MovieDetails/styled.js:21]: Both background and shadows are black, but with no poster we wont render this whole div so there wont be a problem.

File: src/features/people/PersonDetails/index.js (hunk 4 above)
Comment (golosgit) [reply src/features/people/PersonDetails/index.js:43]: Position of the rating corrected.

File: src/features/movies/MovieDetails/index.js (hunk 1 above)
Comment (golosgit) [reply src/features/movies/MovieDetails/index.js]: All errrors corrected.

Comment (maxnatalia) [reply src/features/movies/MovieDetails/index.js]: For 320px is still one column cast people in movie details instead of two.

**Review by maxnatalia — COMMENTED** [review]
We have still problem with responsiveness,  320px and with 280px and 1024(nest hub).

File: src/features/movies/MovieDetails/index.js (hunk 1 above)
Comment (golosgit) [reply src/features/movies/MovieDetails/index.js]: Fixed

**Review by maxnatalia — APPROVED** [review]
Ok, you can merge

**Pull request merged**